constrain/library/ExteriorLightingControlDaylightOff.py

```diff
@@ -28,44 +40,55 @@
 ```
 
 ### Data requirements
-- is_sun_up: flag indicating whether the sun is up; data can be either a boolean (True or False), or numeric boolean (0 or 1)
-- daylight_sensed: Amount of daylight sensed by a photocell type sensor; unit should be consistent with `daylight_setpoint`
-- daylight_setpoint: Setpoint or threshold below which daylight is not sufficient and exterior lighting is required
-- total_lighting_power: reported total lighting power (not the design total lighting power)
+
+- flag_sun_up: Sun position flag
+  - Data Value Unit: binary
+  - Data Point Affiliation: Environmental conditions
+
+- value_daylight: Measured daylight level
+  - Data Value Unit: illuminance
+  - Data Point Affiliation: Lighting control
+
+- value_daylight_setpoint: Daylight threshold
+  - Data Value Unit: illuminance
+  - Data Point Affiliation: Lighting control
+
+- power_light_total: Lighting power
+  - Data Value Unit: power
+  - Data Point Affiliation: Lighting system
```

Comment:
To me, the original descriptions of these data points make more sense.

Reply:
Sure! I changed to the original description.